_hacks/43.md

```diff
@@ -1,5 +1,20 @@
 ---
 hack_number: 43
 date: 2023-11-25
-upcoming: true
 ---
+
+Rolling into the winter season, this was a refreshing, mostly morning, hack. We did pack a whole lot into those early hours, though, and bread, as per usual, dominated the hackday.
+
+## Things people did
+
+-   **Ryan** led the day off right with some tasty bread tales, and then proceeded to fix a lot of people's hacks, reminiscent of an earlier-period Sound Guy. Oh yes, and he also showed off his OBS skills, recording us and then playing lots of picture-in-picture video, which was actually quite helpful for Adam. He then brought down his own harddrive with not only the hidden, but the _still hidden_ files. There was dabbling in Elixir, to much merryment of the crowd.
+
+-   **Joe** Was an additional bread hacker, undertaking some freshly mixed sourdough baking, and also doing some coursework, which he vaguely hinted at, but then neglected to spill that piping hot tea on _what_ exactly that means!
+
+-   **Panda** was barely convinced to not use Oracle Cloud and will instead move all the infra to CloudRun in GCP. He also helped Ben develop some QR code stuff, with a culmination of triggering a MITM attack on themselves.
+
+-   **Ben** Also showed up for the first half of the day, and immediately tried to show us confidential information. Thankfully, cooler heads prevailed upon him to only show us the non-confidential stuff, so we are safe from lawsuits...but for how long!??!!? He also recommended a dope podcast, that Adam is now listening to. He also forgot his own marketing for the hackday project (QR-TX codes), and had to be reminded what his project was called _multiple times_.
+
+-   **Hugh** will either move out of the country or in some other way obfuscate the source of his multi-country income streams. He also put Irish tourism board Open Data into Datasette and now there's a website where you can find all the castles in Carlow (a surprising number: https://irish-attractions.vercel.app/tags/Castle/counties/Carlow) and all the monastaries in Mayo (I'm beginning to suspect that there's a bug because this is so many monastary related tourist attractions: https://irish-attractions.vercel.app/tags/Monastery/counties/Mayo). Plan your Irish holiday at https://irish-attractions.vercel.app/.
```

Comment:
If *only* they were income streams :P

```suggestion
-   **Hugh** put Irish tourism board Open Data into Datasette and now there's a website where you can find all the castles in Carlow (a surprising number: https://irish-attractions.vercel.app/tags/Castle/counties/Carlow) and all the monastaries in Mayo (I'm beginning to suspect that there's a bug because this is so many monastary related tourist attractions: https://irish-attractions.vercel.app/tags/Monastery/counties/Mayo). Plan your Irish holiday at https://irish-attractions.vercel.app/.
```

Reply:
The feds don't watch github pr comments, thankfully 😆